packages/accounts-app/src/Accounts/AccountOverviewDetailed.tsx

```diff
@@ -92,10 +92,10 @@ export function AccountOverviewDetailed (props: Props) {
       .getOrElse(false);
 
     const isStashValidating = fromNullable(allStashes)
-      .map(allStashes => allStashes.includes(new AccountId(currentAccount)))
+      .map(allStashes => allStashes.includes(createType('AccountId', currentAccount)))
       .getOrElse(false);
 
-    const accountType = fromNullable(stakingInfo).map(stakingInfo => new AccountId(currentAccount) === stakingInfo.controllerId ? 'controller' : 'stash');
+    const accountType = fromNullable(stakingInfo).map(stakingInfo => createType('AccountId', currentAccount) === stakingInfo.controllerId ? 'controller' : 'stash');
```

Comment:
Does the `===` here actually work? Since both are objects?

Assuming you have an Codec instance, it has and `.eq` that checks against a variety, i.e. in . the case of `AccountId`, it will check agsinst -

- Other AccountId instances
- ss-58 addresses
- hex publickeys
- Uint8Array publickeys